google-cloud-dataproc/src/test/java/com/google/cloud/dataproc/v1/it/ITSystemTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.dataproc.v1.it;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.dataproc.v1.Cluster;
+import com.google.cloud.dataproc.v1.ClusterConfig;
+import com.google.cloud.dataproc.v1.ClusterControllerClient;
+import com.google.cloud.dataproc.v1.ClusterControllerSettings;
+import com.google.cloud.dataproc.v1.CreateClusterRequest;
+import com.google.cloud.dataproc.v1.RegionName;
+import com.google.cloud.dataproc.v1.WorkflowTemplate;
+import com.google.cloud.dataproc.v1.WorkflowTemplateName;
+import com.google.cloud.dataproc.v1.WorkflowTemplateServiceClient;
+import com.google.common.collect.Lists;
+import java.util.Iterator;
+import java.util.List;
+import java.util.UUID;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSystemTest {
+  private static ClusterControllerClient clusterClient;
+  private static WorkflowTemplateServiceClient workflowClient;
+
+  private static final String PROJECT_ID = ServiceOptions.getDefaultProjectId();
+  private static final String REGION = "global";
+  private static final String ID = UUID.randomUUID().toString();
+  private static final int VERSION = 1;
+  private static final String CLUSTER_NAME =
+      "test-cluster-" + UUID.randomUUID().toString().substring(0, 8);
+  private static final String CLUSTER_REGION = "us-central1";
+  private static final ClusterControllerSettings.Builder CLUSTER_CONTROLLER_SETTINGS =
+      ClusterControllerSettings.newBuilder().setEndpoint("us-central1-dataproc.googleapis.com:443");
```

Comment:
Is there a reason this needs to be a builder vs. a built instance?

Also, can we add a comment on why we need to configure a regional endpoint?